# Deconvulation-Methods

## a. Input Bulk RNAseq + Pure Expressions

1. FARDEEP (2019): 
  Publication: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6522071/
  Tool: https://github.com/YuningHao/FARDEEP.git
2. CiberSort (2018)
  Publication: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5895181/
  Tool: https://cibersort.stanford.edu/
3. MySort (2018) <- Based on CiberSort
  Publication: https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2069-6
  Tool: https://testtoolshed.g2.bx.psu.edu/repository?repository_id=6e9a9ab163e578e0&changeset_revision=e3afe097e80a
4. DeMixT (2019):
  Publication: https://www.cell.com/iscience/pdf/S2589-0042(18)30187-1.pdf
  Tool: https://github.com/wwylab/DeMixTallmaterials
5. IsoPure (2013):
  Publication: https://genomemedicine.biomedcentral.com/articles/10.1186/gm433
  Tool: https://static-content.springer.com/esm/art%3A10.1186%2Fgm433/MediaObjects/13073_2013_429_MOESM1_ESM.ZIP
6. IsoPureR (2019):
  Publication: https://www.nature.com/articles/s41467-019-10929-z.pdf
  Tool: https://cran.r-project.org/web/packages/ISOpureR/index.html
7. quanTIseq (2019):
  Publication: https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-019-0638-6#Sec25
  Tool: https://icbi.i-med.ac.at/software/quantiseq/doc/
8. ConsensusTME (2019):
  Publication: https://www.biorxiv.org/content/biorxiv/early/2018/10/11/437533.full.pdf
  Tool: https://github.com/cansysbio/ConsensusTME
9. xCell
  Publication:https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1349-1
  Tool: http://xcell.ucsf.edu/
10. MCP-counter:
  Publication: https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1070-5
  Tool: https://github.com/ebecht/MCPcounter
11. ABsolute Immune Signal (ABIS) deconvolution (2019)
  Publication: https://www.sciencedirect.com/science/article/pii/S2211124719300592
  Tool: https://github.com/giannimonaco/ABIS
12. brm-phylo (2019?):
  Publication: Phylogenies Derived from Matched Transcriptome Reveal the Evolution of Cell Populations and Temporal Order of Perturbed  Pathways in Breast Cancer Brain Metastases (Email Adrian)
  Tool: https://github.com/brm-phylo/brm-phylo


## b. Input scRNAseq + Bulk RNAseq

1. MuSic (2019)
  Publication: https://www.nature.com/articles/s41467-018-08023-x
  Tool: https://github.com/xuranw/MuSiC
2. MuSic (2019)
  Publication: https://www.nature.com/articles/s41467-018-08023-x
  Tool: https://github.com/xuranw/MuSiC
3. BSEQ-sc (2016)
  Publication: https://www.ncbi.nlm.nih.gov/pubmed/27667365
  Tool: http://github.com/shenorrlab/bseq-sc
  
## c. Input scRNAseq + Reference Pure Expressions  
1. SingleR (2019):
  Publication: https://www.nature.com/articles/s41590-018-0276-y
  Tool: https://github.com/LTLA/SingleR 
2. DESCEND (2018):
  Publication: https://www.pnas.org/content/pnas/115/28/E6437.full.pdf
  Tool: https://github.com/jingshuw/descend

## Reviews and Benchmark Studies:
1. Comprehensive evaluation of cell-type quantification methods for immuno-oncology  
   Publication: https://www.biorxiv.org/content/biorxiv/early/2018/11/14/463828.full.pdf
   Supplementary: https://grst.github.io/immune_deconvolution_benchmark/index.html
2. Comprehensive Benchmarking and Integration of Tumour Microenvironment Cell Estimation Method
   Publication: https://www.biorxiv.org/content/biorxiv/early/2018/10/11/437533.full.pdf
3. Quantifying tumor-infiltrating immune cells from transcriptomics data
   Publication: https://link.springer.com/content/pdf/10.1007%2Fs00262-018-2150-z.pdf
4. RNA-Seq Signatures Normalized by mRNA Abundance Allow Absolute Deconvolution of Human Immune Cell Types
   Publication: https://www.cell.com/cell-reports/pdf/S2211-1247(19)30059-2.pdf
